UKHomeOffice/sgmr-service
# NMSW 1367

## AC
User should be redirected to a "Verify Email Address" page when they're attempting to log in with a non-verified email address. Once they've clicked the button they should receive an email to verify their email address

----

## To test
 - Create an account
 - Do not verify the account
 - Log in to the app using the new account
   - App should redirect to the /resend-verification-email URL
 - Click the "Resend Verification Email" button
   - Should receive another verification email
 - Verify your email address using the link in the email
 - Log in to the app
   - Should be taken to landing page


----

## Notes

N/A
